src/renderer/index.ts

```diff
@@ -24,7 +24,7 @@ function prep_state(klass){
   }
 }
 
-game.state.add('boot'       , prep_state(StatesBoot))
+game.state.add('boot', new StatesBoot());
```

Comment:
Ah this appears to fix the scoping issue. for boot. Uncertain as to why.

Reply:
Because I don't have to create separate objects from the state instance.